WcaOnRails/app/webpacker/lib/utils/edit-schedule.js

```diff
@@ -9,65 +11,33 @@ export function toDegrees(coord) {
   return coord / 1e6;
 }
 
-const currentElementsIds = {
-  venue: 0,
-  room: 0,
-  activity: 0,
-};
-
 function withNestedActivities(activities) {
   if (activities.length === 0) return [];
+
   return [
     ...activities,
-    ...withNestedActivities(_.flatMap(activities, 'childActivities')),
+    ...withNestedActivities(activities.flatMap((activity) => activity.childActivities)),
   ];
 }
 
-export function initElementsIds(venues) {
-  // Explore the WCIF to get the highest ids.
-  const maxId = (objects) => _.max(_.map(objects, 'id')) || 0;
-  const rooms = _.flatMap(venues, 'rooms');
-  const activities = _.flatMap(rooms, (room) => withNestedActivities(room.activities));
-  currentElementsIds.venue = maxId(venues);
-  currentElementsIds.room = maxId(rooms);
-  currentElementsIds.activity = maxId(activities);
-}
-
-export function newVenueId() {
-  currentElementsIds.venue += 1;
-  return currentElementsIds.venue;
-}
+const maxId = (objects) => _.max(objects.map((wcifObj) => wcifObj.id)) || 0;
```

Comment:
```suggestion
const maxIdOrZero = (objects) => _.max(objects.map((wcifObj) => wcifObj.id)) || 0;
```

Reply:
Changed. I didn't commit your suggestion because of follow-up changes that would only be propagated through my IDE, not through GitHub Review suggested changes.